interoperability/definitions.qmd

```diff
@@ -48,31 +67,25 @@ Copernicus Data Space Ecosystem for openEO or Ellip for OGC Processes.
 
 It is important to note that APEx itself is not an algorithm hosting platform; rather, it promotes the reuse of existing
 platforms. A key property of algorithm hosting platforms is their long-term sustainability, beyond the lifetime of a
-typical EO application project. This ensures that algorithms can still be executed after the project ends.
+typical EO project. This ensures that algorithms can still be executed after the project ends.
 
 The algorithm hosting platform has an important responsibility to ensure the continued availability of hosted
 algorithms. This responsibility is detailed in the requirements below, highlighting that the selection of the platform
 affects properties such as cost, performance, stability, and the amount of computing resources available to run the
 algorithm. Compliance with these requirements does not necessarily imply a high overall quality level across all
-aspects, ensuring that EO application projects retain a sufficient degree of freedom in selecting their preferred
+aspects, ensuring that EO projects retain a sufficient degree of freedom in selecting their preferred
 platform.
 
 ## Actors
 
-### EO Application Project
-A project that is executed within the ESA Earth Observation (EO) 'Future-EO1 Segment-2' programme, specifically the
-Applications Element of the ‘Earth Science for Society’ block. The Applications Element aims to pioneer innovative and
-reliable EO applications and solutions in support of international environment and development policies (e.g. UN 2030
-agenda on Sustainable Development). Generally, the more applied science projects, commonly those not directly addressing
-Earth system science objectives, are considered application projects.
-
-The ESA EO application project refers to the consortium that is responsible for building the application algorithm. ESA
-will indicate if a specific application project must comply with APEx guidelines. When compliance is required, the
+### EO Project
+The EO project refers to the consortium that is responsible for building the application algorithm. ESA
+will indicate if a specific project must comply with APEx guidelines. When compliance is required, the
```

Comment:
Should we also state that projects can also ask for support or be compliant even if not explicitly required?

"""
ESA EO projects that do not have an explicit compliance requirement are also eligible to receive support. The ESA APEx support can boost project impact, so projects are encouraged to inquire with their ESA technical officer about the possibilities.
"""

Reply:
Yes, indeed, this is good to add.